qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -4,6 +4,7 @@
     type = qemu_guest_agent
     gagent_name = "org.qemu.guest_agent.0"
     gagent_install_cmd = "yum install -y qemu-guest-agent"
+    gagent_uninstall_cmd = "yum -y remove qemu-guest-agent"
```

Comment:
`yum remove` will remove the dependency, which might be used by other packages, `rpm -e --nodeps` might be safer.